packages/extension-api/src/extension-api.d.ts

```diff
@@ -2295,4 +2295,16 @@ declare module '@podman-desktop/api' {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     export function setValue(key: string, value: any, scope?: 'onboarding'): void;
   }
+
+  export interface CliToolOptions {
+    id: string;
+    name: string;
+    displayName: string;
+    description: string;
+    images: ProviderImages;
```

Comment:
this type is not defined in the .d.ts file

Reply:
It is here https://github.com/containers/podman-desktop/blob/312dcd2bf988f001c52c6ae2ff73a0cf1c1deeca/packages/extension-api/src/extension-api.d.ts#L316